samples/connectedhomeip/lock/README.rst

```diff
@@ -0,0 +1,186 @@
+.. _connectedhomeip_lock_sample
+
+Connected Home over IP: Lock
+############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Connected Home over IP Lock sample demonstrates the usage of 
+`CHIP <https://www.connectedhomeip.com/>`_ to build a door lock device with one
```

Comment:
Links should not be used directly, but added to the ncs/nrf/doc/nrf/links.txt and referenced to its name.

Reply:
Done

Reply:
I meant replacing all links in file, like these in the "Commissioning and remote control" section. Sorry that I didn't write it clearly.